### ISSUE TITLE
Getting error 'Stack ... does not exist' after updating to v1.18.1

### ISSUE THREAD

**Issue by mziethen** [aws/copilot-cli#3571]
Hello,
I started a copilot project with version 1.17.0 a few weeks ago. I have deployed 5 services without any problems. Today, I updated to version 1.18.1, created a new service with ```copilot svc init --name ...``` and wanted to deploy it. After copilot updated my staging environment to v1.9.0 I got the following error when I try ```copilot svc deploy --name ... --env staging```:

````
✘ Proposing infrastructure changes for stack <app_name>-staging-<svc_name>
✘ deploy service <svc_name> to environment staging: deploy service: check if changeset is empty: create change set copilot-3ea3048f-d3d7-4531-ae79-36a7b619e953 for stack <app_name>-staging-<svc_name>: ValidationError: Template error: instance of Fn::GetAtt references undefined resource CustomResourceRole
	status code: 400, request id: 6922ff6f-c789-4971-8f29-b6ca17346218: describe change set copilot-3ea3048f-d3d7-4531-ae79-36a7b619e953 for stack <app_name>-staging-<svc_name>: ValidationError: Stack [<app_name>-staging-<svc_name>] does not exist
	status code: 400, request id: 47937578-6294-4382-a3df-ad222ecbd4f4: describe stack: describe stack events for stack <app_name>-staging-<svc_name>: ValidationError: Stack [<app_name>-staging-<svc_name>] does not exist
	status code: 400, request id: abfdf3f6-2bc7-4ed8-84a1-1bd3e625e8aa

**Comment by dannyrandall** [comment]
Hey @mziethen! Thank you for reporting this issue.

To help me look into this, could you share your manifest file that is causing this issue?

**Comment by mziethen** [comment]
Hey @dannyrandall! Yes, of course. The image I use is in a private dockerhub repository. This is my manifest file:

```yml
# The manifest for the "route" service.
# Read the full specification for the "Load Balanced Web Service" type at:
#  https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/

# Your service name will be used in naming your resources like log groups, ECS services, etc.
name: route
type: Load Balanced Web Service

# Distribute traffic to your service.
http: false
nlb:
  port: 8002

# Configuration for your containers and service.
image:
  location: <private/dockerhub_image>
  credentials: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/dockerhub
  # Port exposed through your container to route traffic to it.
  port: 8002

entrypoint: "/run.py --mode main --config /conf/config.json"

cpu: 1024       # Number of CPU units for the task.
memory: 3072    # Amount of memory in MiB used by the task.
count:          # Number of tasks that should be running in your service.
  range:
    min: 1
    max: 3
  cpu_percentage: 65

exec: true     # Enable running commands in your container.

storage:
  volumes:
    data:
      efs: true
      path: /data
      read_only: false
```

**Comment by dannyrandall** [comment]
Thanks! I'm able to recreate the issue - Although I'm also getting the same error using v1.17.0.

It looks like the bug occurs when you use autoscaling along with an NLB - as a temporary workaround while we work on a fix, could you see if disabling autoscaling (set `count` to a fixed number) makes `svc deploy` work?

Sorry about the inconvenience!🥺 Thank you for reporting this bug!

**Comment by mziethen** [comment]
@dannyrandall thanks for your help with this 🙏 when I'm setting ```count``` to a fixed number it's working.

**Comment by huanjani** [comment]
This fix is now released in v1.19.0: https://github.com/aws/copilot-cli/releases/tag/v1.19.0! 🎉
